EIPS/eip-5792.md

```diff
@@ -1,390 +1,269 @@
-of the transaction tracked by the dapp. Dapps often link to the block explorer for the returned transaction hash, but in
-these cases the transaction hash is wrong and the link will not work.
-
-Dapps need a better interface for sending batches of function calls from the user's wallet so they can interact with
-wallets without considering the differences between wallet implementations. These methods are backwards compatible with
-existing EOA wallets. EOA wallets may send bundles of function calls as individual transactions. The goal of the new
-interface is to be a more stable and flexible interface that enables a better user experience, while wallet
-implementations evolve over time.
+The current methods used to send transactions from the user wallet and check their status are `eth_sendTransaction` and `eth_getTransactionReceipt`.
+
+The current methods used to send transactions from the user wallet and check their status do not meet modern developer demands and cannot accomodate new transaction formats. Even the name–`eth_sendTransaction`–is an artifact of a time when nodes served as wallets. 
+
+Today, developers want to send batch transactions, which many smart accounts can support atomically. Developers also want to make use of features afforded by new transaction formats, like paymasters in [ERC-4337](./eip-4337.md) transactions. `eth_sendTransaction` offers no way to do these things. 
+
+In updating to a new set of `wallet_` RPCs, our main goals are to enforce a clean separation of wallet and app concerns, enable developers to make use of things like paymasters and batch transactions, and to create a clear way for more features to be added over time with minimal coordination.
 
 ## Specification
 
-Three new JSON-RPC methods are added. Dapps may begin using these methods immediately, falling back
-to `eth_sendTransaction` and `eth_getTransactionReceipt` when they are not available.
+Four new JSON-RPC methods are added. Apps may begin using these methods immediately, falling back to `eth_sendTransaction` and `eth_getTransactionReceipt` when they are not available.
+
+We define one capability related to call execution atomicity.
+
+### `wallet_getCapabilities`
 
-### `wallet_sendFunctionCallBundle`
+RPC for an application to receive information about the capabilities that a connected wallet supports (e.g. batch transactions, paymaster communication).
 
-Requests that the wallet deliver a group of function calls on-chain from the user's wallet.
+The key of each item per chain is the name of a capability and the value can be of any shape. Capabilities are returned per chain because wallets may support different capabilities across chains.
+
+We expect the community to align on the definition of capabilities in separate ERCs.
+
+#### `wallet_getCapabilities` RPC Specification
+
+```typescript
+type GetCapabilitiesParams = [];
+
+type GetCapabilitiesResult = Record<`0x${string}`, <Record<string, any>>; // Hex chain id
```

Comment:
We should be avoiding the `any` type here. Ideally it should be narrowed, and if not, `unknown`.